src/GraphQL/Schemas/Primary/Mutations/SetWalletAccountMutation.php

```diff
@@ -104,4 +103,25 @@ public function resolve(
 
         return $walletService->update($wallet, ['public_key' => SS58Address::getPublicKey($args['account'])]);
     }
+
+    /**
+     * Get the validation rules.
+     */
+    protected function rules(array $args = []): array
+    {
+        return [
+            'account' => [
+                'bail',
+                'filled',
+                new ValidSubstrateAccount(),
+                function (string $attribute, mixed $value, Closure $fail): void {
+                    if (Wallet::where('public_key', SS58Address::getPublicKey($value))
+                        ->withoutGlobalScope('tenant')->exists()
```

Comment:
```suggestion
                        ->withoutGlobalScopes()->exists()
```
Should we run without any scopes, rather than just this one specifically?  Just mindful of including direct references to other packages, if can avoid that it would be preferable.